### DISCUSSION TITLE
野生的鲨鱼希人

### DISCUSSION THREAD

**Discussion by MiManchi-netizen** [share121/inter-knot#2550] [其他]
家人们，抓到一只野生的鲨鱼希人
![20240815224014](https://github.com/user-attachments/assets/c16a30eb-e00e-4c13-878c-c9d6f48014b9)

**Comment by liusugong** [comment]
实则猎人往往以猎物的形式出现

**Comment by NekoWise** [comment]
合影费付一下，谢谢

**Comment by hyperion-netizen** [comment]
好看！
